jfix-stdlib-batching/src/main/java/ru/fix/stdlib/batching/BatchingManagerMetricsProvider.java

```diff
@@ -0,0 +1,83 @@
+package ru.fix.stdlib.batching;
+
+import org.jetbrains.annotations.NotNull;
+import ru.fix.aggregating.profiler.IndicationProvider;
+import ru.fix.aggregating.profiler.ProfiledCall;
+import ru.fix.aggregating.profiler.Profiler;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.Queue;
+
+public class BatchingManagerMetricsProvider {
+
+    private final String batchManagerId;
+    private final Profiler profiler;
+    private final String batchProcessorThreadAwaitMetric;
+    private final String batchManagerOperationsAwaitMetric;
+    private final List<String> operationQueueSizeIndicatorsList = new ArrayList<>();
+    private ProfiledCall batchProcessorThreadAwaitProfiledCall;
```

Comment:
Здесь нужно задокументировать политику работы с классом - предполагается однопоточный последовательный доступ?

Reply:
Смушает, что объект переиспользуется. У нас не должен при старте возвращаться новый call, который должен закрыть клиентский код?

Reply:
Yes we assuming only single threaded usage because class instance used in single threaded BatchingManagerThread

ProfiledCall recreated everytime profiler.start() called